### ISSUE TITLE
fontawesome 5

### ISSUE THREAD

**Issue by fabioformosa** [madoar/angular-archwizard#134]
Has someone used fontawesome 5 in `aw-wizard-step` ?

I've imported
```
<link rel="stylesheet" 
              href="https://use.fontawesome.com/releases/v5.1.1/css/all.css" 
              integrity="sha384-O8whS3fhG2OnA5Kas0Y9l3cfpmYjapjI0E4theH4iuMD+pLhbf6JI0jIMfYcK3yZ"
              crossorigin="anonymous">

...

<aw-wizard-step stepTitle="Package Selection" 
            [navigationSymbol]="{ symbol: '&#xf005;', fontFamily: 'Font Awesome 5 Free' }">

```

It fails! I see a square at the place of right icon

**Comment by madoar** [comment]
Hmm, this seems to be linked to https://github.com/FortAwesome/Font-Awesome/issues/11946, can you verify this?

If this is the case, I believe you need to apply custom css to the navigation bar, in which you specify the required `font-weight`.
Can you test, if this solves the issue?

**Comment by fabioformosa** [comment]
No, it's not a `font-weight` matter. I tried.
I suppose the problem is in fontFamily of navigationSymbol attribute.
I report what I've found by browser inspection

```
aw-wizard-navigation-bar.horizontal.large-filled-symbols ul.steps-indicator li:after {
...
content: attr(step-symbol);
...
}
```
No font-family found. If I add manually `font-family: 'Font Awesome 5 Free'`, icon renders fine.

Am I using in the right way `navigationSymbol`?

**Comment by madoar** [comment]
Ok, I think I understand the issue.
This should be fixed together with PR #129.

**Comment by elebitzero** [comment]
Hi,

I also cannot get Font Awesome 5 to work with aw-wizard.  This is using latest fontawesome 5.4.1 (free).
I thought I could just change the font family to 'Font Awesome 5 Free' but it doesn't work.   I tried setting the font-weight: 900 on the aw-wizard but that didn't do anything.

The last comment by @madoar seems indicate the issues is fixed, but is it?   I checked the angular-archwizard-demo, and I notice that is using font-awesome 4.7.1 version.   Maybe I'll downgrade to that version so I can get it to work.

**Comment by madoar** [comment]
The fix has been introduced in the development branch.
The changes will be released in version `4.0.0`, which still needs a little bit of time until it is released.
If you require font awesome support earlier you can build the development version yourself for now, and then switch to the official release when it is released.

**Comment by elebitzero** [comment]
@madoar, thanks.  For now, I downgraded back to font-awesome 4.7.1, and it is working.  I'll upgrade when new version of aw-wizard is released.

**Comment by madoar** [comment]
I've just released `angular-archwizard` version `4.0.0` to [npm](https://www.npmjs.com/package/angular-archwizard)! Feel free to give it a try!
